--- src/build_sources/Makefile.am~	Sun May 13 07:02:48 2001
+++ src/build_sources/Makefile.am	Sat Apr 20 21:33:00 2002
@@ -6,8 +6,8 @@
 gensourcedir=$(top_builddir)/src/gnome--
 
 gnomemmhwithdirs = $(GEN_H_FILES:.gen_h=.h)
-gnomemmh = $(gnomemmhwithdirs)
-gnomemmc = $(gnomemmh:.h=.cc) gnome--.cc
+#gnomemmh = $(gnomemmhwithdirs)
+gnomemmc = $(gnomemmhwithdirs:.h=.cc) gnome--.cc
 
 gnomemmgenh = $(patsubst %.gen_h,$(srcdir)/../%.gen_h,$(gnomemmsources))
 
